src/Sulu/Bundle/ContentBundle/Tests/Functional/Document/SyncronizationManagerTest.php

```diff
@@ -13,6 +13,8 @@
 
 use Sulu\Bundle\ContentBundle\Document\PageDocument;
 use Sulu\Bundle\TestBundle\Testing\SuluTestCase;
+use PHPCR\PropertyType;
+use Sulu\Bundle\ContentBundle\Document\RouteDocument;
 
 class SyncronizationManagerTest extends SuluTestCase
```

Comment:
This tests the specific requirements of Sulu and is therefore coupled to the Sulu configuration. There should be another test which tests the manager with an independent subject (i.e. a test Document).